_posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Apache ServiceComb Meetup -Shanghai 2018.10.12 Report (PPT)"
+lang: cn
+ref: Apache-ServiceComb-Meetup-2018-Shanghai-Report
+permalink: /cn/docs/Apache-ServiceComb-Meetup-2018-Shanghai-Report/
+
+excerpt: "Apache ServiceComb Meetup -Shanghai 2018.10.12 Report (PPT)"
+last_modified_at: 2018-10-29T17:18:43+08:00
+author: Dean Lee
+tags: [microservice, Meetup]
+redirect_from:
+  - /theme-setup/
+---
+![1](/assets/images/hc2018/1.jpeg)
+
+**《Apache之道》**    
+
+[PDF下载](/assets/slides/20181012/EN/1_Apache_HuaweiConnect_Presentation_20181012.pdf)

```

Comment:
Could you change the link name of "1_Apache_HuaweiConnect_Presentation_20181012.pdf" to the correct one？
You can just use the topic name for it.